rtp/jitter.go

```diff
@@ -30,7 +30,7 @@ func HandleJitter(h Handler) Handler {
 	out := make(chan []*rtp.Packet, 10)
 	handler := &jitterHandler{
 		h:   h,
-		buf: jitter.NewBuffer(audioDepacketizer{}, jitterMaxLatency, out, nil),
+		buf: jitter.NewBuffer(audioDepacketizer{}, jitterMaxLatency, out),
```

Comment:
There is an `out` in the `jitterHandler` struct also. Is that needed? Sorry, asking based on super limited view here.

Reply:
oh, it is needed in `run` 👍

Reply:
Sorry, what do you mean?